### PR TITLE
docs(connector): google connector zh-cn

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

docs(connector): google connector zh-cn

## Testing

Tested in connector details page.

### CODE REVIEW

File: packages/connector-google/README.md [hunk 1]
```diff
@@ -1,7 +1,33 @@
-# Google
+# Google connector
 
 The Google connector provides a succinct way for your application to use Google’s OAuth 2.0 authentication system.
 
+Google 连接器为您的应用程序提供了一种接入 Google OAuth 2.0 身份验证系统的简单方式。
```
Comment (gao-sun) [inline packages/connector-google/README.md:5]: 你

File: packages/connector-google/README.md [hunk 2]
```diff
@@ -56,3 +82,60 @@ Now you should have the Google OAuth 2.0 consent screen configured.
...
+- 根据页面提示，填写「OAuth 同意屏幕」相关表单。
+- 点按「保存并继续」以进行后续操作。
+
+#### 配置权限「范围」
+
+- 点击「添加或移除范围」按钮，然后在弹出的抽屉页面中选择 `../auth/userinfo.email`、 `../auth/userinfo.profile` 和 `openid`，然后点击「更新」以完成配置。
+- 根据你的需求填写页面上的相关的表单。
+- 点按「保存并继续」以进行后续操作。
+
+#### 添加「测试用户」（仅「User Type」设置为「外部」时需要）
```
Comment (gao-sun) [inline packages/connector-google/README.md:116]: is this the translation in google API console? 「外部」「OAuth 同意屏幕」

Comment (xiaoyijun) [reply packages/connector-google/README.md:116]: Yes, this follows the Chinese version of the Google API Console.

File: packages/connector-google/README.md [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+  - [Configure your consent screen](#configure-your-consent-screen)
+    - [Configure and register your application](#configure-and-register-your-application)
+    - [Edit app registration](#edit-app-registration)
+      - [Config OAuth consent screen](#config-oauth-consent-screen)
+      - [Config scopes](#config-scopes)
+      - [Add test users (External user type only)](#add-test-users-external-user-type-only)
+  - [Obtain OAuth 2.0 credentials](#obtain-oauth-20-credentials)
+    - [Config types](#config-types)
+  - [References](#references)
+- [Google 链接器](#google-链接器)
```
Comment (demonzoo) [inline packages/connector-google/README.md:19]:
```suggestion
- [Google 连接器](#google-连接器)
```
```suggestion
- [Google 链接器](#google-链接器)
```

File: packages/connector-google/README.md [hunk 4]
```diff
@@ -1,7 +1,33 @@
...
+  - [在 Google API 控制台中创建一个项目](#在-google-api-控制台中创建一个项目)
+  - [配置 OAuth 同意屏幕](#配置-oauth-同意屏幕)
+    - [注册和配置应用](#注册和配置应用)
+    - [修改应用注册](#修改应用注册)
+      - [设置「OAuth 同意屏幕」](#设置oauth-同意屏幕)
+      - [配置权限「范围」](#配置权限范围)
+      - [添加「测试用户」（仅「User Type」设置为「外部」时需要）](#添加测试用户仅user-type设置为外部时需要)
```
Comment (demonzoo) [inline packages/connector-google/README.md:26]: Oh, shoot. I have to trim these special characters in AC guide to support these internal toc links

File: packages/connector-google/README.md [hunk 5]
```diff
@@ -56,3 +82,60 @@ Now you should have the Google OAuth 2.0 consent screen configured.
 
 ## References
 * [Google Identity: Setting up OAuth 2.0](https://developers.google.com/identity/protocols/oauth2/openid-connect#appsetup)
+
+# Google 链接器
```
Comment (demonzoo) [inline packages/connector-google/README.md:86]:
```suggestion
# Google 连接器
```

File: packages/connector-google/README.md (hunk 2 above)
Comment (gao-sun) [reply packages/connector-google/README.md:116]: please double check if "User Type" shows as English in the Chinese version before merging

**Review by gao-sun — APPROVED** [review]

File: packages/connector-google/README.md [hunk 6]
```diff
@@ -56,3 +82,60 @@ Now you should have the Google OAuth 2.0 consent screen configured.
...
+
+#### 配置权限「范围」
+
+- 点击「添加或移除范围」按钮，然后在弹出的抽屉页面中选择 `../auth/userinfo.email`、 `../auth/userinfo.profile` 和 `openid`，然后点击「更新」以完成配置。
+- 根据你的需求填写页面上的相关的表单。
+- 点按「保存并继续」以进行后续操作。
+
+#### 添加「测试用户」（仅「User Type」设置为「外部」时需要）
+
+- 点击 「+ ADD USERS」添加测试用户，以遍这些用户能在应用的发布状态为”测试时“能访问该应用。
```
Comment (demonzoo) [inline packages/connector-google/README.md:118]: Does it have Chinese translation for this `+ ADD USERS` button?

Comment (demonzoo) [inline packages/connector-google/README.md:118]:
```suggestion
- 点击 「+ ADD USERS」添加测试用户，以便这些用户能在应用的发布状态为”测试时“能访问该应用。
```

File: packages/connector-google/README.md [hunk 7]
```diff
@@ -56,3 +82,60 @@ Now you should have the Google OAuth 2.0 consent screen configured.
...
+现在你已经配置好了 Google 的「OAuth 同意屏幕」。
+
+## 获取 OAuth 2.0 凭据
+
+- 在左侧「API 和服务」的菜单中，点按「凭据」选项。
+- 在「凭据」页面内的上部菜单栏中，点击「+ 创建凭据」，然后选择「OAuth 客户端 ID」。
+- 在「创建 OAuth 客户端 ID」页面内，将「应用类型」设置为「Web 应用」。
+- 填写相关的应用基本信息。
+- 在「已获授权的 JavaScript 来源」中，点击「+ 添加URI」，该 URI 值为你所部署的 Logto 服务的地址。此处这个值应为 `${your_logto_origin}`。例如：`https://logto.dev`。
+- 在「已获授权的重定向 URI」中，点击「+ 添加URI」以添加「已获授权的重定向 URI」。「已获授权的重定向 URI」将在用户成功使用 Google 帐号成功登录 Logto 后将用户重定向回你的应用。此处这个值应为 `${your_logto_origin}/callback/google-universal`。例如：`https://logto.dev/callback/google-universal`.
```
Comment (demonzoo) [inline packages/connector-google/README.md:130]:
```suggestion
- 在「已获授权的重定向 URI」中，点击「+ 添加URI」以添加「已获授权的重定向 URI」。「已获授权的重定向 URI」将在用户成功使用 Google 帐号成功登录 Logto 后将用户重定向回你的应用。此处这个值应为 `${your_logto_origin}/callback/google-universal`。例如：`https://logto.dev/callback/google-universal`.
```
```suggestion
- 在「已获授权的重定向 URI」中，点击「+ 添加URI」以添加「已获授权的重定向 URI」。「已获授权的重定向 URI」将在用户使用 Google 帐号成功登录 Logto 后将用户重定向回你的应用。此处这个值应为 `${your_logto_origin}/callback/google-universal`。例如：`https://logto.dev/callback/google-universal`.
```

File: packages/connector-google/README.md [hunk 8]
```diff
@@ -1,7 +1,33 @@
-# Google
+# Google connector
```
Comment (demonzoo) [inline packages/connector-google/README.md:1]:
```suggestion
# Google Connector
```

File: packages/connector-google/README.md (hunk 6 above)
Comment (xiaoyijun) [reply packages/connector-google/README.md:118]: Yes

Comment (xiaoyijun) [reply packages/connector-google/README.md]: image

File: packages/connector-google/README.md (hunk 8 above)
Comment (xiaoyijun) [reply packages/connector-google/README.md:1]: This aligns with other readme files.

**Pull request merged**